packages/grz-pydantic-models/tests/test_metadata.py

```diff
@@ -69,6 +69,77 @@ def test_wgs_trio_no_vcf(version):
     GrzSubmissionMetadata.model_validate_json(json.dumps(metadata))
 
 
+@pytest.mark.parametrize(
+    "version",
+    TESTED_VERSIONS,
+)
+def test_wgs_tumor_germline_missing_dna(version):
+    """
+    Ensure that both tumor and germline DNA lab data are required for WGS tumor-germline submissions.
+    """
+    metadata_str = (
+        importlib.resources.files(resources)
+        .joinpath("example_metadata", "wgs_tumor_germline", f"v{version}.json")
+        .read_text()
+    )
+
+    ### tumor+germline
+
+    # delete germline DNA
+    metadata = json.loads(metadata_str)
+    del metadata["donors"][0]["labData"][0]
+
+    # missing germline DNA should fail
+    with pytest.raises(ValidationError):
```

Comment:
Perhaps we should match the actual ValidationError message, to make sure the thing that fails validation is what we expect it to be ;D